lib/Metadata.php

```diff
@@ -82,6 +82,23 @@ public function getFieldMappings()
     }
 
     /**
+     * Return the mapping for the named field.
+     *
+     * @return array
+     */
+    public function getFieldMapping($name)
+    {
+        if (!isset($this->fieldMappings[$name])) {
+            throw new \InvalidArgumentException(sprintf(
+                'Field mapping "%s" not known in metadata for document type "%s". Known mappings: "%s"',
+                $name, $this->alias, implode('", "', array_keys($this->fieldMappings))
+            ));
+        }
+
+        return $this->fieldMappings[$name];
+    }
+
+    /**
      * @return string
      */
     public function getClass()
```

Comment:
Not related to this PR, but this class contains a wrong phpdoc containing `ReflectionClass` instead of `\ReflectionClass`.